AzureFunctionsSDK/AzureFunctionsSDK.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.2.1" />
```

Comment:
Yup! The challenge is that, at compile-time, we cannot test that this external package will work with any given version of the worker.

I wonder if there's any amount of dependency isolation we could do, at the worker level, to solve this. At the very least, our release process should have an explicit check to make sure our dependencies are compatible.

Reply:
I don't worry about this dependency, chances are we don't need to do anything special about it. We may want to keep the SDK reference reasonably fresh, but I'm quite certain there are many modules built with old SDK references, and they keep somehow just working on the latest PowerShell, without the need to re-build and re-release them. This interface is very stable.

Reply:
The main issue is with dependency conflicts - since the worker also uses this package, if the worker loads one version, and this library loads another, you get runtime errors. So, we either need to build in some kind of dependency isolation to the library, or make sure that the worker and the library's versions are always the same.